source/features/link-issues-pr-user-profile.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import elementReady from 'element-ready';
+import issueIcon from 'octicon/issue-opened.svg';
+import pullRequestIcon from 'octicon/git-pull-request.svg';
+
+async function init(): Promise<false | void> {
+	const repositoryTab = await elementReady('[data-filterable-for="your-repos-filter"]');
+	if (!repositoryTab) {
+		return;
+	}
+
+	for (const repository of select.all('[itemprop="owns"]', repositoryTab)) {
+		// Remove the help wanted issues
+		select('[href*="issues?q=label%3A%22help+wanted%"]', repository)?.remove();
+
+		const repositoryUrl = select<HTMLAnchorElement>('[itemprop="name codeRepository"]')!.href;
+		// Need to use previousSibling since the updated text is not in an element
+		select('relative-time', repository)!.previousSibling!.before(
+			<a
+				className="muted-link mr-3"
+				href={repositoryUrl + '/issues?q=is%3Aissue+is%3Aopen'}
+			>
+				{issueIcon()}
+			</a>,
+			<a
+				className="muted-link mr-3"
+				href={repositoryUrl + '/pulls?q=is%3Aissue+is%3Aopen'}
+			>
+				{pullRequestIcon()}
+			</a>
+		);
+	}
+}
+
+features.add({
+	id: __featureName__,
+	description: 'Adds a link to the issues and pull on the user profile repository tab.',
+	screenshot: 'https://user-images.githubusercontent.com/16872793/78670305-8ab3c780-78ab-11ea-9c17-4d07f124192e.png'
+}, {
+	include: [
+		features.isUserProfile
+	],
+	exclude: [
+		features.isOrganizationProfile
+	],
+	load: features.nowAndOnAjaxedPages,
```

Comment:
This can't use `now` because it's applied on the whole page, not just one element. Skip the `elementReady` too